about.html

```diff
@@ -0,0 +1,142 @@
+     </ul>
+          <div class="hamburger">
+        <div class="bar"></div>
+        <div class="bar"></div>
+        <div class="bar"></div>
+        </div>
+    </nav>
+    <div class="wrapper">
+        
+      <div class="intro">
+        
+        <h3 class="title-about">"Hello Freshers!"</h3>
+        <h1 class="about-h1">Mobile first design strategy</h1>
+        <p class="about-info">
+          Lorem ipsum dolor sit amet consectetur, adipisicing elit. Impedit
+          magnam odit praesentium excepturi eveniet, dolor dolorem ex incidunt
+          perferendis
+        </p>
+        <p class="caption-about">2020/30/10 Join us for better knowledge</p>
+        <p class="caption-about1">@Kabul University Afghanistan</p>
+      </div>
+      <div class="main-program">
+        
+        <h2 class="main-program1">Main program</h2>
+        <div class="main">
+        <div class="lecturesection">
+         <img src="images/program_icon_01.png" alt="">
+          <h3 class="lecture">Lecture</h3>
+          <p class="lecture-p">Lorem ipsum dolor sit amet consectetur</p>
+        </div>
+        <div class="lecturesection">
+          <img src="images/program_icon_02.png" alt="">
+          <h3 class="lecture">UI Exhibition</h3>
+          <p class="lecture-p">Lorem ipsum dolor sit</p>
+        </div>
+        <div class="lecturesection">
+          <img src="images/program_icon_03.png" alt="">
+          <h3 class="lecture">Forum</h3>
+          <p class="lecture-p">Lorem ipsum dolor sit amet consectetur.</p>
+        </div>
+        <div class="lecturesection">
+         <img src="images/program_icon_04.png" alt="">
+          <h3 class="lecture">Workshop</h3>
+          <p class="lecture-p">Lorem ipsum dolor</p>
+        </div>
+        <div class="lecturesection">
+        <img src="images/program_icon_05.png" alt="">
+          <h3 class="lecture">Ignit</h3>
+          <p class="lecture-p">Lorem ipsum dolor sit amet consectetur</p>
+        </div>
+        </div>
+        <a class="see" href="">See the whole program</a>
+       
+        <a class="button" href="">Get your Tickets</a>
+      </div>
+        <h2 class="feature-speaker">featured Speakers</h2>
+        <hr class="line">
+      <div class="featured">
+       
+
+        </div>   
+          
+        <div class="more">
+          <h4 class="more-speaker">more </h>
+          <a href="" class="more-image"><img  class="more-img" src="images/Disabled.svg" alt=""></a>
+        </div>
+
+      <div class="partner">
+        <h3 class="partners">Partner companies</h3>
+           <hr class="line1">
+        <div class="companies">
+          <h6 class="mozilla">Mozilla</h6>
+        <h6 class="google">Google</h6>
+        <h6 class="youtube">YouTube</h6>
+        <h6 class="meta">Meta</h6>
+        <h6 class="airbnb">airbnb</h6>
+
+        </div>
+      </div>
+      <footer><p class="footer">@copyright UI-Ux designers</p></footer>
```

Comment:
**[OPTIONAL]** - Since these requirements have been skipped by the previous reviewers, I am marking them as optional, but I would advise that you implement them as they would really help you to make your project look great and more presentable. Your footer for both the index and about page does not match the design template. Replicating the design template as closely as possible will give confidence to recruiters in your ability to replicate Figma designs.

Home footer (Your project):
![home footer current](https://user-images.githubusercontent.com/73607512/190496807-7d74ae4a-163a-4f2e-92c1-1958a5ee6e79.PNG)

Home footer(Design template):
![home footer design](https://user-images.githubusercontent.com/73607512/190496899-590fd3d2-59e1-4b96-b3d0-c87c5f03e683.PNG)

About footer (Your project):
![footer current](https://user-images.githubusercontent.com/73607512/190496992-e83eff14-180c-44c5-96bb-ba495d1a60b9.PNG)

About footer(Design template):
![about footer design](https://user-images.githubusercontent.com/73607512/190497274-711cc20c-29fa-456a-afcf-94d1d6b1c3f9.PNG)